.github/workflows/valgrind-checks.yaml

```
@@ -9,23 +9,24 @@ concurrency:
+        compiler_profile: [clang-libstdcxx, gcc-libstdcxx]
     runs-on: ubuntu-latest
     steps:
-      - name: Maximize build space
-        uses: easimon/maximize-build-space@v10
-        with:
-          overprovision-lvm: true
-          remove-dotnet: true
-          remove-android: true
-
       - name: Check out repository code
         uses: actions/checkout@v3
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)